iotedgedev/envvars.py

```diff
@@ -98,7 +98,7 @@ def load(self, force=False):
 
                 self.BYPASS_MODULES = self.get_envvar("BYPASS_MODULES", default="")
                 self.CONTAINER_TAG = self.get_envvar("CONTAINER_TAG", default="")
-                self.RUNTIME_TAG = self.get_envvar("RUNTIME_TAG", default="1.0")
+                self.RUNTIME_TAG = self.get_envvar("RUNTIME_TAG", default="")
```

Comment:
>"" [](start = 74, length = 2)

Why empty default value?

Reply:
It has to do with the sequence of calls that creates the template files, which you want to leave as $(RUNTIME_TAG) when they're being generated to be swapped later down the line. The default is being specified somewhere else.